config/logstash.yml

```diff
@@ -113,3 +113,12 @@
 #
 # Where to find custom plugins
 # path.plugins: []
+#
+# ------------ Slow log thresholds ---------
+#
+# An slow log threshold is a time limit for an operation that when is overcome it will
+# be reported at the related log file. Time is seconds.
+#
+# The last building block of the component is the log level (default: warn)
+#
+# filters.dns.getaddress.warn: 5
```

Comment:
fair point, I initially thought this problem could be addressed with documentation, but having a global switch makes also sense. We can always add this information as operational context.